scripts/hid_configurator/README.rst

```diff
@@ -133,7 +133,7 @@ Configuring device runtime options
 ==================================
 
 The script can pass the configuration values to the linked FW module using the ``config`` command.
-Use the following syntax to display list of configurable modules:
+Use the following syntax to display list of modules that can have device runtime options configured:
```

Comment:
Can we leave it as `configurable modules`? I think it's simpler and the configured options actually belong to an application module that is part of the firmware, not to a whole device

Reply:
Also `ble_bond` is a `configurable module`, but it does not actually configure the device runtime options (you can trigger Bluetooth scanning or peer erase using `config` command for this module)

Reply:
OK, that's fine.